Operaciones Básicas con Matrices

gerswanb opened the pull request:
**Mejora en la estética del programa.**

gerswanb committed:
Operaciones Básicas con Matrices

LEOITOO56 commented:
_**Verificación exitosa, se mejoro la apariencia del programa y funciona a la perfección.**_

LEOITOO56 committed:
Update GRUPO DE PROGRAMACION.cpp Actividad 2

LEOITOO56 committed:
Merge branch 'main' into gerswanb-obcm-v1

gerswanb committed:
Operaciones Básicas con Matrices

Se soluciono errores, y el programa funciona a la perfección, con una mejor interfaz.
